databases.star

```diff
@@ -1,53 +1,72 @@
+# We support both local and remote Postgres databases within our Kurtosis-CDK package
+# When 'USE_REMOTE_POSTGRES' is False, service automatically creates all CDK databases locally
+# When 'USE_REMOTE_POSTGRES' is True, service is created just as a helper for param injection across pods
+# When 'USE_REMOTE_POSTGRES' is True, all state is stored on your preconfigured remote Postgres instances
+USE_REMOTE_POSTGRES = False
+
+# When 'USE_REMOTE_POSTGRES' is True, replace 'POSTGRES_HOSTNAME' with your master database IP/hostname
+POSTGRES_HOSTNAME = "127.0.0.1"
+
+# Mostly static params unless user has specialized postgres configuration
 POSTGRES_IMAGE = "postgres:16.2"
 POSTGRES_SERVICE_NAME = "postgres"
 POSTGRES_PORT = 5432
+
+# Below 'POSTGRES_MASTER_' params only apply when 'USE_REMOTE_POSTGRES' is False
 POSTGRES_MASTER_DB = "master"
 POSTGRES_MASTER_USER = "master_user"
 POSTGRES_MASTER_PASSWORD = "master_password"
 
-CDK_DATABASES = {
+# When 'USE_REMOTE_POSTGRES' is True, update following credentials to match your remote postgres DBs
+# It is recommended users keep existing DB names and usernames for stability
+# This way, users can also leverage our 'reset_postgres.sh' script,
+# Which automatically wipes all CDK databases and reapplies proper db permissions
+# TO DO: add env var support for credentials
+TRUSTED_DATABASES = {
```

Comment:
Not sure about the renaming here. I feel like event, pool, prover and state dbs are zkevm-node databases (not sure about trusted here). And same thing for `PERMISSIONLESS_DATABASES`, I find that `DATABASES` was better but still, it's not a great name...

Reply:
could be wrong, based naming on our kurtosis-cdk architecture diagram: https://github.com/0xPolygon/kurtosis-cdk/blob/main/docs/img/architecture.png as original var names bit ambiguous

curious your and @praetoriansentry's point of view, what var names do you think would be clearer here ?

Reply:
after speaking w/ @praetoriansentry , its my understanding var names are viable for now

going to proceed with merging @leovct